packages-exp/analytics-exp/src/api.ts

```diff
@@ -216,7 +216,7 @@ export function setAnalyticsCollectionEnabled(
   ).catch(e => logger.error(e));
 }
 /**
- * Sends analytics event with given `eventParams`. This method
+ * Sends Google Analytics event with given `eventParams`. This method
```

Comment:
Suggest "Sends a Google Analytics..."

Here and below.

Reply:
Done.